### ISSUE TITLE
Table Column Filters Are Not Keyboard Accessible

### ISSUE THREAD

**Issue by mcook103** [ant-design/ant-design#44774]
### What problem does this feature solve?

Some visually impaired users use their keyboard to navigate through applications by using the tab, enter, and arrow keys.  The antd table has some functionality that is keyboard accessible (column sorting, row expanding, etc.) but the filter icons on each column header are not accessible.  It would be helpful if the user could tab to the filter icon, press enter or the down arrow, and have the column's filter menu displayed.

### What does the proposed API look like?

As mentioned above, some functionality is keyboard accessible "out of the box".  The keyboard accessibility of the column filters should also work "out of the box".

**Comment by afc163** [comment]
Duplicate of #37508
